Assets/_BForBoss/_Core/Scripts/UserInterface/EntityCounterViewBehaviour.cs

```diff
@@ -0,0 +1,30 @@
+using Perigon.Entities;
+using TMPro;
+using UnityEngine;
+
+namespace BForBoss
+{
+    public class EntityCounterViewBehaviour : MonoBehaviour
+    {
+        [SerializeField] private TMP_Text _entityCounterLabel = null;
+        private int _entitiesDestroyed = 0;
+
+        public void Initialize(LifeCycleManager lifeCycleManager)
+        {
+            lifeCycleManager.OnLivingEntitiesAmountChanged += HandleLivingEntitiesAmountChanged;
+        }
+
+        public void Reset()
+        {
+            _entitiesDestroyed = 0;
+            _entityCounterLabel.text = _entitiesDestroyed.ToString();
+        }
+
+        private void HandleLivingEntitiesAmountChanged()
+        {
+            _entitiesDestroyed++;
+            _entityCounterLabel.text = _entitiesDestroyed.ToString();
+        }
```

Comment:
It might make more sense to keep track of the number killed within the manager, and just access it here, just in case we end up wanting to use it elsewhere. I think that way you could also remove the coupling between the manager and individual lifecycles that way, too, for Martin's point